compiler/src/iree/compiler/GlobalOptimization/PropagateLinalgTranspose.cpp

```diff
@@ -29,6 +32,11 @@
 
 namespace mlir::iree_compiler::GlobalOptimization {
 
+static llvm::cl::opt<bool> clOnlySinkTransposes(
+    "iree-global-opt-only-sink-transposes",
+    llvm::cl::desc("Forces transpose propagation to only sink them."),
+    llvm::cl::init(false));
+
```

Comment:
I'm not sure if I have cycles to review it, so I only point it out for now. This is not used, can we remove it?

Reply:
oh oops, this was leftover from sdxl, good catch!